1-exercises/C-object-methods/exercise-2.js

```diff
@@ -15,7 +15,7 @@ let myPets = {
   rabbit: "Bugs",
 };
 
-let petNames =
+let petNames = Object.values(myPets)
   /* todo */
```

Comment:
Well done Shayan 😊
I think the exercie requires to return the values in an array.